example_test.go

```diff
@@ -67,6 +67,55 @@ func ExampleClient_WrapCreateDEK() {
 	// wrappedDek is your WDEK, keep this and pass it to Unwrap to get back your DEK when you need it again
 }
 
+func ExampleClient_WrapCreateDEKWithAlias() {
```

Comment:
We should add ExampleClient_WrapCreateKeyAlias() and ExampleClient_WrapCreateDEKWithAlias() & ExampleClient_UnWrapCreateDEKWithAlias()